### PR TITLE
CORE-480: Start the EWM event handler on `ewmStart()` not on `ewmConnect()`.

### PR DESCRIPTION
**Pull request by EBGToo**
CWM will call ewmCreate(...), set up `cwm` and `cwm->wallet`, and then call `ewmStart()` to generate EWM events.  It might be that CWM doesn't need to generate its own CWM events (in cwmCreate()) as the EWM events, once flowing, will generate CWM events(?).

### CODE REVIEW

File: ethereum/ewm/BREthereumEWM.c [hunk 1]
```diff
@@ -653,6 +663,27 @@ ewmDestroy (BREthereumEWM ewm) {
...
+
+    // Start the EWM thread
+    eventHandlerStart(ewm->handler);
+}
+
+extern void
+ewmStop (BREthereumEWM ewm) {
+    // Stop the alarm clock
+    alarmClockStop (alarmClock);
+
```
Comment (michael-brd) [inline ethereum/ewm/BREthereumEWM.c]: Devil's advocate (not sure myself if we want this), but should stop imply disconnect? Almost like how on Android the lifecycle is onCreate->onStart->onResume->onPause->onStop->onDestroy (so, ewmCreate->ewmStart->ewmConnect->ewmDisconnect->ewmStop->ewmDestroy)?

Comment (EBGToo) [reply ethereum/ewm/BREthereumEWM.c]: Maybe it is worse.... disconnect => stop (which it shouldn't); stop !=> disconnect (which it should).

File: ethereum/ewm/BREthereumEWM.c [hunk 2]
```diff
@@ -653,6 +663,27 @@ ewmDestroy (BREthereumEWM ewm) {
     free (ewm);
 }
 
+/// MARK: - Start/Stop
+
+extern void
+ewmStart (BREthereumEWM ewm) {
+    // Start the alarm clock.
+    alarmClockStart(alarmClock);
```
Comment (michael-brd) [inline ethereum/ewm/BREthereumEWM.c]: This is the process-wide alarm clock, ya? Don't think we want to be doing this, as it has repercussions on any other handler using a timeout, if I am understanding correctly.

Comment (EBGToo) [reply ethereum/ewm/BREthereumEWM.c]: CORE-489

File: ethereum/ewm/BREthereumEWM.c (hunk 1 above)
Comment (EBGToo) [reply ethereum/ewm/BREthereumEWM.c]: Updated: `ewmStop()` calls `ewmDisconnect()`.  

Added a number of TODOs - for example, noting that start() and stop() don't check on the current state (which they arguably should).

**Comment by EBGToo** [comment]
Can we move this to 'approved' understanding that we've more work to do on the sequencing but, with merging, to allow the testing of ETH to continue?

**Pull request merged**